src/pages/index.js

```diff
@@ -185,40 +185,30 @@ export default function Home() {
           </section>
         )}
       </main>
-      <div className={clsx('hero hero--secondary', styles.heroBanner)}>
+      <div className={clsx('hero hero--secondary boo', styles.heroSecondRow)}>
         <div className="container">
-          <h2>Core concepts</h2>
-          <div className="text--center">
-            <a href={useBaseUrl("img/concept-map.svg")}>
-              <img className={styles.featureImage} src={useBaseUrl("img/concept-map.svg")} alt="concepts" />
-            </a>
-          </div>
+          <h2>🍎 Core concepts</h2>
           <p>Review some of Temporal's core concepts and building blocks.</p>
-          <div className={styles.buttons}>
-            <Link
-            className={clsx(
-              'button button--outline button--primary button--md',
-              styles.getStarted,
-            )}
-            to={useBaseUrl("/docs/concept-overview")}>
-            Learn more
-            </Link>
-          </div>
+          <ul>
+            <li><Link to={useBaseUrl("/docs/concept-overview")}> Overview </Link></li>
+            <ul>
+              <li><Link to={useBaseUrl("/docs/concept-workflows")}> Workflows </Link></li>
```

Comment:
I think these should all be in the same unordered list and change Overview to Introduction, i.e:
- Introduction
- Workflows
- Activities
- Workers
- More... etc